WordPress/Classes/Extensions/NSAttributedString+StyledHTML.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+
+extension NSAttributedString {
+    /// Creates an `NSAttributedString` with the styles defined in `attributes` applied.
+    /// - parameter htmlString: The string to be styled. This can contain HTML
+    ///    tags to markup sections of the text to style, but should not be wrapped
+    ///    with `<html>`, `<body>` or `<p>` tags. See `HTMLAttributeType` for supported tags.
+    /// - parameter attributes: A collection of style attributes to apply to `htmlString`.
+    ///    See `HTMLAttributeType` for supported attributes.
+    ///
+    ///    **Notes:**
```

Comment:
You can use `- note:` here

https://developer.apple.com/library/ios/documentation/Xcode/Reference/xcode_markup_formatting_ref/Note.html